ERROR    - 2024-07-22 21:14:50,714. В модуле "task_1" в строке 036 функция "__init__()" в 1721672090.7140331 секунд записала сообщение: Вы ввели неверный атрибут - last_name. Ваш ввод - 12
ERROR    - 2024-07-22 21:14:50,714. В модуле "task_1" в строке 040 функция "__init__()" в 1721672090.7140331 секунд записала сообщение: Вы ввели неверный атрибут - first_name. Ваш ввод - 13
ERROR    - 2024-07-22 21:14:50,714. В модуле "task_1" в строке 044 функция "__init__()" в 1721672090.7140331 секунд записала сообщение: Вы ввели неверный атрибут - patronymic. Ваш ввод - 134
ERROR    - 2024-07-22 21:14:50,714. В модуле "task_1" в строке 048 функция "__init__()" в 1721672090.7140331 секунд записала сообщение: Вы ввели неверный атрибут - age. Ваш ввод - sds
ERROR    - 2024-07-22 21:14:50,714. В модуле "task_1" в строке 085 функция "__init__()" в 1721672090.7140331 секунд записала сообщение: Вы ввели неверный атрибут - position. Ваш ввод - 24
